### ISSUE TITLE
License not found :)

### ISSUE THREAD

**Issue by yaminoma** [smorimoto/smorimoto#2]
I can't find any license in your profile, what kind of license is that?

**Comment by smorimoto** [comment]
Indeed, I hesitated to add a license for software to this repository, and it has not been added yet. Licenses like CC are probably a good idea. I always welcome PR.
